### ISSUE TITLE
Add Angular

### ISSUE THREAD

**Issue by birthdaysgift** [fenya123/bingin#18]
**As an Architect I would like to have Angular as a main framework for frontend application to be able to develop extensible SPA with full typescript support.**

Before we start development we need to set up tools we're going to use and check that everything works together fine.

In the scope of this story we need to add Angular in our frontend repository and commit **minimal**  working angular setup.

Since angular's `ng new` adds a lot of stuff we don't need at this moment, we should clean it up and keep only things that we actually need.

We also don't want to have all compiler checks which are set by `ng new` by default since we're going to have a separate story for configuring compilers/linters checks.

**Tasks**

- [x] https://github.com/fenya123/acih-frontend/issues/1
